CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+*November 19, 2021*
+
+## v0.6.2
```

Comment:
is that change in gRPC (`QueryAccountRequest`) breaking -- i.e. will the v0.6.1 binary still be able to use that query against v0.6.2 node?

Reply:
I think it is ok. Ethermint does not provide any query command and cronos query commands do not use it

Reply:
https://github.com/crypto-org-chain/ethermint/commit/9f1478e3d35c538dce3c26971e3f5e8f550c7685#diff-324e7e7d0d1ab912bb60777c369af7304487e0a49727c3b9cf600ae9ef7865caL6
Is it this one? it looks like the only protobuf change.

Reply:
yeah, to my limited understanding, it's only adding a few fields, so most protobuf tooling should be fine (deserialisers in the old version would ignore it / put it in unknown fields: https://developers.google.com/protocol-buffers/docs/proto3#unknowns )

Reply:
The only api change is the `TraceTx ` and `TraceBlock`, they are not exposed through CLI I think.